tests/integration/test_integration.py

```diff
@@ -7,10 +7,12 @@
 import os
 from os.path import abspath
 from pathlib import Path
+from typing import List
 
 import hvac  # type: ignore[import-untyped]
```

Comment:
(not related to this PR) Do we still need to ignore this import for static analysis?

Reply:
Broken out separately: https://github.com/canonical/vault-operator/pull/84